specifyweb/specify/migrations/0002_geo.py

```diff
@@ -33,63 +32,75 @@ class Migration(migrations.Migration):
     ]
 
     def create_default_collection_types(apps, schema_editor):
-        # Create default collection types for each discipline
-        for discipline in Discipline.objects.all():
+        # Create default collection types for each collection, named after the discipline
+        for collection in Collection.objects.all():
+            discipline = collection.discipline
             discipline_name = discipline.name
-            if CollectionObjectType.objects.filter(name=discipline_name).exists():
-                continue
-            cot = CollectionObjectType.objects.create(
+            cot, created = CollectionObjectType.objects.get_or_create(
                 name=discipline_name,
-                isdefault=True,
-                collection = Collection.objects.filter(discipline=discipline).first(),
-                taxontreedef=discipline.taxontreedef,
+                collection=collection,
+                taxontreedef=discipline.taxontreedef
```

Comment:
Just a quick note here (which I'm sure I've brought up multiple times 😅), but we'll shortly have a different way of handling TaxonTreeDefs. That is, allowing more than one TaxonTreeDef per Discipline (#5016). 

We'll need to think about how we want to resolve this change with the current code. Whether we want to just use the TaxonTreeDef which is referenced on the Discipline, or have some other way to resolve the "default".

Reply:
We discussed this and decided to have a foreign key to the discipline in the `taxontreedef` (and the other discipline-scoped trees).

Reply:
👍 
Great! In that case, #5016 can be closed. There are a few branches waiting on this change! 
(Complete functionality for #5036, #5091, and many many more changes which need to be done to completely support multiple trees of a type at a scope)

Reply:
UPDATE: Working on this, fixing some migration errors.

Reply:
@melton-jason check out the updated migration file for setting the default discipline for each tree def.